neos/neos-development-collection
# BUGFIX: Adjust assetsource icon color on hover and active

The icons are always white and when the asset source is active and on a small viewport we switch the background also to white. So the icons are not visible anymore.

This patch will adjust the color of the icon via filter functions.
As we are not able to use the fill rule.

Fixes: #3417

**What I did**
We now use the css filter to modify the color of the icons.
```
filter: invert(.5) sepia(1) saturate(5) hue-rotate(175deg);
```

**How to verify it**

1. Install a second asset source and go to the media module.
2. Shrink the viewport to less than 767 pixels width
3. Check if the icon of the active asset source is visible


https://user-images.githubusercontent.com/1014126/134674449-e293e16b-0a03-434b-9914-9434701fa56b.mp4



**Checklist**
